UCLALibrary/voyager-archive
# SYS-1038: Display brief holdings with bib record

Implements [SYS-1038](https://jira.library.ucla.edu/browse/SYS-1038).

This PR adds:
* When a bib record displays, brief information for each linked holdings record now displays below the bib record
* Each line of brief holdings data is linked to the full holdings record display
* The full holdings record display is linked to its bib record via the 004 field
* A new database view and Django model, `MfhdSummaryView`, provides the data.  There's a new migration for this model.
* A new URL pattern, `marc_display/<str:marc_type>/<int:record_id>`, supports this and will be used later on PO line and invoice line displays

Testing:
* Pull and checkout branch `SYS-1038/brief_holdings_on_bib`
* (Re)start docker-compose system to apply new migration
* Run `sql_support/initialize_database.sh` to refresh database objects
* Search for bib 9411434
* Scroll down to see holdings
* Click links to navigate between records

Brief holdings info, with link to full holdings record:
![9411434_bib_brief_holdings](https://user-images.githubusercontent.com/2213836/197893085-ec2dc9e6-809d-431e-85e8-4e28a02de726.png)

Full holdings, with 004 link to bib record:
![9411434_holdings_link_to_bib](https://user-images.githubusercontent.com/2213836/197893137-bd37eb33-0990-4db7-84b2-ca6a4fb83def.png)
